### PR TITLE
feat(electron): app menu item and hotkey for creating new page

### PR DESCRIPTION
**Pull request by SevenOutman**
Press <kbd>Command + N</kbd> on macOS or <kbd>Ctrl + N</kbd> on Windows/Linux to create new page in current workspace

https://user-images.githubusercontent.com/8225666/236797359-7743def1-60de-4c3e-9fff-ebb410245a9b.mov

### CODE REVIEW

File: apps/electron/layers/preload/preload.d.ts [hunk 1]
```diff
@@ -3,4 +3,16 @@
...
+     * Adds a listener for "New Page" action in application menu
+     *
+     * @returns A cleanup function that removes the listener when called
+     */
+    onNewPage: (callback: (e: Electron.IpcRendererEvent) => void) => () => void;
+    /**
+     * Enable/disable "New Page" action in application menu
+     */
+    toggleNewPageAction: (enabled: boolean) => void;
+  };
```
Comment (himself65) [inline apps/electron/layers/preload/preload.d.ts]: I think this should be moved into apis? 
/cc @pengx17 

BTW, there're more and more APIs on the electron side; I think we should restart MessagePort PR(https://github.com/toeverything/AFFiNE/pull/2080) to separate them

Comment (SevenOutman) [reply apps/electron/layers/preload/preload.d.ts:17]: > The implementation should be refactored after #2122 is merged. Can you @SevenOutman please make changes based on it?

Sure. I'll keep an eye on that PR and update my changes once it's merged.

Comment (himself65) [reply apps/electron/layers/preload/preload.d.ts]: It merged now

File: apps/electron/layers/main/src/application-menu.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import { app, Menu } from 'electron';
+
+import { subjects } from './events';
+
+// Unique id for menuitems
+const MENUITEM_NEW_PAGE = 'affine:new-page';
+
+export function createApplicationMenu() {
+  const isMac = process.platform === 'darwin';
```
Comment (pengx17) [inline apps/electron/layers/main/src/application-menu.ts:9]: You can use `isMacOS()` in util.ts

File: apps/electron/layers/main/src/application-menu.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    },
+    {
+      role: 'help',
+      submenu: [
+        {
+          label: 'Learn More',
+          click: async () => {
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
+            const { shell } = require('electron');
+            await shell.openExternal('https://electronjs.org');
```
Comment (pengx17) [inline apps/electron/layers/main/src/application-menu.ts:113]: Can you change the URL to https://affine.pro/?

File: apps/electron/layers/main/src/application-menu.ts (hunk 2 above)
Comment (SevenOutman) [reply apps/electron/layers/main/src/application-menu.ts]: I've updated it.

File: apps/electron/layers/main/src/application-menu.ts (hunk 3 above)
Comment (SevenOutman) [reply apps/electron/layers/main/src/application-menu.ts]: Good idea. Updated the link!

**Review by pengx17 — APPROVED** [review]
Works great, thanks!

**Pull request merged**